tests/python/gaia-ui-tests/gaiatest/resources/htmlandcssbook/code-samples/chapter-17/introduction.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
thought about later expanding the test to click the link to go there, but until then I can remove this page I suppose